docs/ref/class-based-views/generic-date-based.txt

```diff
@@ -593,7 +593,7 @@ DateDetailView
         from django.views.generic.dates import DateDetailView
 
         urlpatterns = [
-            url(r'^(?P<year>[0-9]+)/(?P<month>[-\w]+)/(?P<day>[0-9]+)/(?P<pk>[0-9]+)/$',
+            url(r'^(?P<year>[0-9]{4})/(?P<month>[-\w]+)/(?P<day>[0-9]+)/(?P<pk>[0-9]+)/$',
```

Comment:
Every other example on [that page](https://docs.djangoproject.com/en/1.8/ref/class-based-views/generic-date-based/) uses `(?P<year>[0-9]{4})`, so I was making it consistent with that. 

I could change all of the examples on the page to use counts for the month and day parameters. I note that `[-\w]+` is a bit different to `[a-z]` since it allows upper and lowercase - I don't know if that's a big issue or not.